contracts/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "compile": "hardhat starknet-compile",
+    "compile": "hardhat starknet-compile ./contracts ./vendor",
```

Comment:
Do we need this `./contracts ./vendor` definition, if it's already defined in `paths.cairoPaths` in the `hardhat.config.ts`?

Reply:
That is because the contracts under ./vendor need to be compile as well and for that I need to give the path during the compilation. The `path.caroPaths` is for the import path because in cairo you can can't have path with `-`into it.